### PR TITLE
CODE RUB: Action variable for testing exception for retrieve all in student registration

### PR DESCRIPTION
**Pull request by deepshri7**

### CODE REVIEW

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/StudentRegistrations/StudentRegistrationServiceTests.Exceptions.RetrieveAll.cs [hunk 1]
```diff
@@ -50,9 +50,15 @@ public void ShouldThrowServiceExceptionOnRetrieveAllStudentRegistrationsWhenExce
                 broker.SelectAllStudentRegistrations())
                     .Throws(exception);
 
-            // when . then
-            Assert.Throws<StudentRegistrationServiceException>(() =>
-                this.studentRegistrationService.RetrieveAllStudentRegistrations());
+
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/StudentRegistrations/StudentRegistrationServiceTests.Exceptions.RetrieveAll.cs:53]: Please remove the extra lines 53 , 61

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/StudentRegistrations/StudentRegistrationServiceTests.Exceptions.RetrieveAll.cs [hunk 2]
```diff
@@ -50,9 +50,15 @@ public void ShouldThrowServiceExceptionOnRetrieveAllStudentRegistrationsWhenExce
...
+            // when
+            Action retrieveAllStudentRegistrationAction = () =>
+                this.studentRegistrationService.RetrieveAllStudentRegistrations();
+
+            // then
+            Assert.Throws<StudentRegistrationDependencyException>(
```
Comment (hassanhabib) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/StudentRegistrations/StudentRegistrationServiceTests.Exceptions.RetrieveAll.cs:59]: This also should `StudentRegistrationServiceException`

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/StudentRegistrations/StudentRegistrationServiceTests.Exceptions.RetrieveAll.cs [hunk 3]
```diff
@@ -50,9 +54,14 @@ public void ShouldThrowServiceExceptionOnRetrieveAllStudentRegistrationsWhenExce
...
-            Assert.Throws<StudentRegistrationServiceException>(() =>
-                this.studentRegistrationService.RetrieveAllStudentRegistrations());
+             // when
+            Action retrieveAllStudentRegistrationAction = () =>
+                this.studentRegistrationService.RetrieveAllStudentRegistrations();
+
+            // then
+            Assert.Throws<StudentRegistrationServiceException>(
+                retrieveAllStudentRegistrationAction);
+
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/StudentRegistrations/StudentRegistrationServiceTests.Exceptions.RetrieveAll.cs:64]: Extra line here can be removed.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

**Comment by ShriHumrudha** [comment]
@deepshri7, please fix the build failure.

**Comment by deepshri7** [comment]
Made new PR which run successfully...

**Comment by ShriHumrudha** [comment]
@deepshri7 Got a conflict.

**Pull request closed without merging**